test/reporter-html/unhandled-rejection.js

```diff
@@ -0,0 +1,35 @@
+// Currently (2017-12-18) only Chrome has implemented `unhandledRejection`
+const IS_CHROME = !!window.chrome;
```

Comment:
`onrejectionhandled in window` might be easier to maintain as a feature test, or are there false positives for that? That way when other browsers land support, or CI will verify them automatically.